### ISSUE TITLE
[BUG] Closing inventories from other plugins throws an error in console

### ISSUE THREAD

**Issue by simondumalski** [snowgears/shopbugs#429]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of bug

Error in console

### Environment

```markdown
- Server Version: Paper version 1.20.6-148-ver/1.20.6@20f5165
- Shop Version: 1.9.0.0
```

### Server startup log

https://paste.gg/p/anonymous/995f2b55bb5d4e569309a9133c9e0a7f

### Shop config files

https://paste.gg/p/anonymous/b3dbe3ffea3f4bd085e92c8c45354c0a

### Error log (if applicable)

https://paste.gg/p/anonymous/ad97238dda544f63a83724c89045324a

### Bug description

Closing any custom inventory from another plugin (such as a crate preview in ExcellentCrates or a menu in DeluxeMenus) throws an error in console upon close

### Steps to reproduce

1. Open a custom inventory
2. Error thrown

### Expected behaviour

No error in console

### Actual behaviour

Error is thrown in console

**Comment by OstlerDev** [comment]
Bug being worked on currently, bugfix will be included in 1.9.0.1 (when released)

**Comment by OstlerDev** [comment]
Fixed in 1.9.0.1
